core/themes/bartik/css/style.css

```diff
@@ -1185,6 +1185,28 @@ ul.action-links li a {
   padding: 0;
 }
 
+ /* -------------- Hero blocks  -------------- */
+
+ .hero-block {
+   font-size: 250%;
+   position: relative;
+   padding: 0;
+ }
+ .hero-block.maintain-aspect-ratio:before {
```

Comment:
the class goes with the markup change. If the markup is not changed then we are not maintaining aspect ratio, so I wanted to do them both in the same place (the template)